components/featuredProjects/ProjectCard.tsx

```diff
@@ -0,0 +1,96 @@
+"use client";
+
+import { Card } from "@nextui-org/react";
+import Image from "next/image";
+import Link from "next/link";
+
+import { arrow } from "@/public";
+
+type projectProps = {
+  name: string;
+  description: string;
+  techStack: string[];
+  image: string;
+  color: string;
+};
+
+const ProjectCard = ({
+  project,
+  index,
+}: {
+  project: projectProps;
+  index: number;
+}) => {
+  return (
+    <Card
+      key={project.name}
```

Comment:
A 'key' prop is not necessary here. It's not part of an array map function. You can remove this.

Reply:
Removed it.